explorer/app/viewModelBuilders/azul/hca-dcp/common/viewModelBuilders.ts

```diff
@@ -820,6 +820,26 @@ function getDCPGeneratedMatricesTableColumns<T>(): ColumnDef<T>[] {
   ];
 }
 
+/**
+ * Returns formatted estimated cell count from the given projects response.
+ * @param projectsResponse - Response model return from projects API.
+ * @param formatFn - Function to format count (optional, e.g. formatCountSize, default is locale string).
+ * @returns formatted estimated cell count.
+ */
+export function getEstimatedCellCount(
```

Comment:
The new function `getEstimatedCellCount` formats the calculated cell count by either locale string or a specified format function like `formatCountSize`.